examples/evaluate.py

```diff
@@ -9,43 +9,98 @@
 from h3ds.dataset import H3DS
 from h3ds.numeric import unidirectional_chamfer_distance
 from h3ds.log import logger
+from h3ds.utils import error_to_color
 
 
 def your_reconstruction_method(images: list, masks: list, cameras: list):
-    return choice([
-        trimesh.primitives.Box(radius=100),
-        trimesh.primitives.Sphere(radius=100),
-        trimesh.primitives.Cylinder(radius=100)
-    ])
+    ''' Returns another H3DS scene randomly to simulate a reconstruction'''
+    scene_id = choice(h3ds_scenes)
```

Comment:
would it make sense to include a seed for randomness?

Reply:
Maybe yes! I didn't do it because it's just an example and the results are not very important.